### PR TITLE
vUSB changes (main page and CLI)

### PR DESCRIPTION
**Pull request by kimsauce**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/dev/cli/virtual-usb/start-server.md [hunk 1]
```diff
@@ -4,53 +4,54 @@ title: "Virtual USB CLI: Start the Server"
 sidebar_label: Start Server
 ---
 
-__Full Example__:
+The `server` command connects your local machine to the Sauce Labs Data Center where your Real Device Cloud private devices are hosted.
```
Comment (wswebcreation) [inline docs/dev/cli/virtual-usb/start-server.md:7]: Can you also mention that this commands needs to be started in a separate terminal and will run the whole time when they connect with different devices. All connection logs, even from the devices, can be find in this terminal

**Review by wswebcreation — COMMENTED** [review]
Hi @kimsauce 

I added some comments, some are essential to change and some are nice to haves, please see what you want to do with it

Grtz

Wim

File: docs/dev/cli/virtual-usb/connect-session.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+id: connect-session
+title: "Virtual USB CLI: Connect to Existing Session"
+sidebar_label: Connect to Session
+---
+
+The `connect` command provides the server with your user credentials and session ID for an existing live test, then connects the device to your local machine. This is one of two ways to start a Virtual USB test; the other is to [start a new test session from the command line](dev/cli/virtual-usb/start-session).
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/dev/cli/virtual-usb/connect-session.md:7]: Thanks! all set

File: docs/dev/cli/virtual-usb/delete-session.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+id: delete-session
+title: "Virtual USB CLI: Delete Session"
+sidebar_label: Delete Session
+---
+
+The `delete` command provides the server with your session ID and user credentials, then deletes your test session from your local machine.
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/dev/cli/virtual-usb/delete-session.md:7]: Yikes, that was a big oversight on my part -- good eye

File: docs/dev/cli/virtual-usb/disconnect-session.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+id: disconnect-session
+title: "Virtual USB CLI: Disconnect Session"
+sidebar_label: Disconnect Session
+---
+
+The `disconnect` command provides the server with your test session ID, then disconnects the real device from your local machine.
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/dev/cli/virtual-usb/disconnect-session.md:7]: perfect!

File: docs/dev/cli/virtual-usb/find-session-id.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+id: find-session-id
+title: "Virtual USB CLI: Find Session ID"
+sidebar_label: Find Session ID
+---
+
+The `sessions` command returns a list of active real device sessions based on the Sauce Labs credentials you enter. This can be useful if you've lost track of your session ID and need to find it to run other commands (i.e., deleting or disconnecting a session).
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/dev/cli/virtual-usb/find-session-id.md:7]: good idea! added

```java title="Basic Example (required flags only)"
java -jar virtual-usb-client sessions --username john.smith --accessKey ab015c1e-xxxx-xxxx-xxxx-xxxxxxxxxxx
```

```java title="Response Example"
07:43:40.929 [main] INFO com.saucelabs.vusb.client.Runner - Runner Version 2.0.0
List of active sessions
d03a1b81-xxxx-xxxx-xxxx-xxxxxxxxxxxx  Samsung Galaxy S10  ANDROID  10
c7729c7a-xxxx-xxxx-xxxx-xxxxxxxxxxxx  iPhone XS           IOS      14.3
e21abb6f-xxxx-xxxx-xxxx-xxxxxxxxxxxx  iPhone SE 2020      IOS      14.3
```

File: docs/dev/cli/virtual-usb/start-session.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+---
+id: start-session
+title: "Virtual USB CLI: Start Test Session"
+sidebar_label: Start Session
+---
+
+The `startSession` command provides the server with your credentials and the name of the device you want to connect to, then launches a brand new live test session via the command line. This is one of two ways to start a Virtual USB test; the other is to [connect to an existing session](dev/cli/virtual-usb/connect-session).
+
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/dev/cli/virtual-usb/start-session.md:8]: done

File: docs/dev/cli/virtual-usb/delete-session.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/dev/cli/virtual-usb/delete-session.md:8]: all set -- thank you for these

File: docs/dev/cli/virtual-usb/start-server.md (hunk 1 above)
Comment (kimsauce) [reply docs/dev/cli/virtual-usb/start-server.md:7]: done! added a short note here, and added a more elaborate note in the main vUSB guide

File: docs/dev/cli/virtual-usb/connect-session.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/dev/cli/virtual-usb/connect-session.md:8]: all set

File: docs/dev/cli/virtual-usb/connect-session.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+## Required
+
+### `--sessionId`
+__Description__: your test session ID.
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/dev/cli/virtual-usb/connect-session.md:12]: done

File: docs/dev/cli/virtual-usb/disconnect-session.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/dev/cli/virtual-usb/disconnect-session.md:8]: all set

File: docs/dev/cli/virtual-usb.md [hunk 11]
```diff
@@ -14,7 +14,7 @@ The command line structure for all vUSB requests is as follows:
 
 :::tip
 
-Use --help to view the full list of available commands and options. The synopsis for each command shows its parameters and their usage.:
+You can also view the full list of vUSB commands and options directly in your CLI by running the `--help` flag. The description for each command shows its parameters and their usage:
 ```java
-java -jar virtual-usb-client-2.0.0.jar --help
+java -jar virtual-usb-client --help
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/dev/cli/virtual-usb.md:19]: will do and will talk to my team about adding the note

**Pull request merged**